pandas_transform/notebooks/dataframe_example.ipynb

```diff
@@ -0,0 +1,2363 @@
+       "      <td>4.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>106</th>\n",
+       "      <td>5002573</td>\n",
+       "      <td>glendon</td>\n",
+       "      <td>lannister</td>\n",
+       "      <td>male</td>\n",
+       "      <td>12</td>\n",
+       "      <td>covid</td>\n",
+       "      <td>emergency dept</td>\n",
+       "      <td>positive</td>\n",
+       "      <td>patient</td>\n",
+       "      <td>18.0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>31.16</td>\n",
+       "      <td>1</td>\n",
+       "      <td>government</td>\n",
+       "      <td>emergency</td>\n",
+       "      <td>0.4</td>\n",
+       "      <td>5.2</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         mrn first_name  last_name  gender  pan_day test_id       clinic_name  \\\n",
+       "27   5002114      azzak      tully    male       10   covid  inpatient ward b   \n",
+       "61   5007240      arryk    mormont    male       11   covid      clinical lab   \n",
+       "91   5011391        zei      umber  female       11   covid      s  care ntwk   \n",
+       "100  5000902       owen   seaworth    male       12   covid    emergency dept   \n",
+       "106  5002573    glendon  lannister    male       12   covid    emergency dept   \n",
+       "\n",
+       "       result  demo_group   age  drive_thru_ind  ct_value  orderset  \\\n",
+       "27   positive     patient   0.9               0     39.55         0   \n",
+       "61   positive  misc adult  21.0               1     39.51         1   \n",
+       "91   positive     patient   0.1               0     39.83         0   \n",
+       "100  positive     patient   0.1               0     32.96         1   \n",
+       "106  positive     patient  18.0               0     31.16         1   \n",
+       "\n",
+       "    payor_group patient_class  col_rec_tat  rec_ver_tat  \n",
+       "27   government   observation          0.6          2.7  \n",
+       "61          NaN           NaN          1.8          4.7  \n",
+       "91   government    outpatient          5.2          4.8  \n",
+       "100  government     emergency          0.3          4.5  \n",
+       "106  government     emergency          0.4          5.2  "
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "positive_tests = covid_testing.loc[covid_testing.loc[:,\"result\"] == \"positive\",:]\n",
+    "positive_tests.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8ead948-a0b0-4f73-b7f8-b8539159177f",
+   "metadata": {},
+   "source": [
+    "In the example above we used the double equals sign `==` to check that the things to the left and the right were the same. This is different from the single equals sign we used at the beginning of this notebook to define `covid_testing`. \n",
```

Comment:
I'd suggest being a bit more explicit here, since this trips a lot of people up, mention that `==` is used for testing for equality and that `=` is for value assignments.